### PR TITLE
Initial version for release

### PR DESCRIPTION
**Pull request by doekenorg**

### CODE REVIEW

**Comment by doekenorg** [comment]
Needs a CHANGELOG.

File: src/TaskListsTrait.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+namespace Kirra\Markdown;
+
+/**
+ * Adds Github's Task Lists support to a parser.
+ * @link https://github.blog/2013-01-09-task-lists-in-gfm-issues-pulls-comments
+ * @since $ver$
+ */
+trait TaskListsTrait {
+	/**
+	 * {@inheritDoc}
```
Comment (pharixces) [inline src/TaskListsTrait.php]: A trait can be included in different files in different situations, inheritDoc would therefore inherit from different things, so you cannot use inheritDoc. Also; the {@inheritDoc} tag cannot be used in a summary, only in the description as inheritance is automatic.

@see https://docs.phpdoc.org/guides/inheritance.html

File: src/TaskListsTrait.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	 *
+	 * This parses a checkbox first. This is because the marker phpdoc only takes one function.
+	 * Make sure to use this trait later then {@see LinkTrait}.
+	 *
+	 * @since $ver$
+	 * @marker [
```
Comment (pharixces) [inline src/TaskListsTrait.php:18]: This is a nonexisting tag according to the phpdoc specification.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+php:
+    - '7.1'
+    - '7.2'
+    - '7.3'
+install: composer install
+after_success:
+    - travis_retry php vendor/bin/php-coveralls
```
Comment (pharixces) [inline .travis.yml]: Add newline to the end of the file.

**Review by pharixces — CHANGES_REQUESTED** [review]

File: src/TaskListsTrait.php (hunk 2 above)
Comment (doekenorg) [reply src/TaskListsTrait.php:18]: This is how it works.

**Review by pharixces — APPROVED** [review]

**Pull request merged**